source/server/server.cc

```diff
@@ -957,8 +955,16 @@ void InstanceImpl::terminate() {
   stats_store_.shutdownThreading();
 
   // TODO: figure out the correct fix: https://github.com/envoyproxy/envoy/issues/15072.
-  Config::GrpcMuxImpl::shutdownAll();
-  Config::NewGrpcMuxImpl::shutdownAll();
+  auto* factory = Config::Utility::getFactoryByName<Config::MuxFactory>(
```

Comment:
nit: this can probably be done in a for loop over the strings, but not sure if it's worth it.